scripts/om.sh

```diff
@@ -55,5 +56,5 @@ if [ -n "${TURN_PASS}" ]; then
 	sed -i "s|p:turnSecret=\"\"|p:turnSecret=\"${TURN_PASS}\"|g" ${CLASSES_HOME}/applicationContext.xml
 fi
 cd ${OM_HOME}
-sudo -u ${DAEMON_USER} ${OM_HOME}/bin/catalina.sh run
+sudo --preserve-env=CATALINA_OPTS -u ${DAEMON_USER} ${OM_HOME}/bin/catalina.sh run
```

Comment:
this needs to pass in the environment variable from the current user into the sudo session. Otherwise above export won't be available to the command.

Reply:
@sebawagner this PR make no sense to me :(
Why `CATALINA_OPTS` should be preserved? who set it? how is this connected with `DATA_DIR`?

Reply:
see line 43. 

export CATALINA_OPTS="-DDATA_DIR=${OM_DATA_DIR}"

=> This export is list when running catalina.sh run. Cause its run as another user.

Reply:
INFO  09-14 09:11:23.641 o.a.o.u.OmFileHelper:109 [main] - Using file locations - OM_HOME: /opt/openmeetings/webapps/openmeetings DATA_DIR: /opt/openmeetings/webapps/openmeetings/data